data/campaigns/World_Conquest/gui/settings_dialog.cfg

```diff
@@ -173,7 +173,7 @@
                         [column]
                             grow_factor = 1
                             [label]
-                                label = "hill_size"
+                                label = _ "hill_size"
```

Comment:
The labels in this dialog might need renaming, rather than just marking as translatable. With its underscore, `hill_size` is the one that I noticed, but many of them seem to need answers to the question "what does this mean?"

Reply:
Yes, the underscore here should be replaced by a space, and some of the other labels may need improvement, perhaps.